portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -39,25 +39,34 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    
+    Integer numComments = Integer.parseInt(request.getParameter("quantity"));
```

Comment:
I would recommend to have some checking here in case someone forgets to specify the parameter `quantity` while making this GET request.

Reply:
Thanks Kazi!